src/adapter/src/catalog/builtin.rs

```diff
@@ -2249,7 +2249,11 @@ pub static MZ_SOURCE_STATISTICS: Lazy<BuiltinSource> = Lazy::new(|| BuiltinSourc
         .with_column("updates_committed", ScalarType::UInt64.nullable(false))
         .with_column("bytes_received", ScalarType::UInt64.nullable(false))
         .with_column("envelope_state_bytes", ScalarType::UInt64.nullable(false))
-        .with_column("envelope_state_count", ScalarType::UInt64.nullable(false)),
+        .with_column("envelope_state_count", ScalarType::UInt64.nullable(false))
+        .with_column(
+            "rehydration_latency_seconds",
```

Comment:
can we make this ms?

Reply:
Yup. I changed it to seconds because that's what we were doing with the upsert metric and because prometheus recommended to use seconds https://prometheus.io/docs/practices/naming/#base-units.

Reply:
Grafana does support `ms` in their UI so I think we should be fine. Let me update this.

Reply:
done!